engines/director/frame.cpp

```diff
@@ -857,6 +857,11 @@ void Frame::renderButton(Graphics::ManagedSurface &surface, uint16 spriteId) {
 
 void Frame::renderText(Graphics::ManagedSurface &surface, uint16 spriteId, Common::Rect *textRect) {
 	TextCast *textCast = (TextCast*)_sprites[spriteId]->_cast;
+	if (textCast == nullptr){
```

Comment:
This formatting is incorrect. I will merge it and then fix. But please, follow our code formatting conventions.